dev/run-dependencies.sh

```diff
@@ -14,6 +14,9 @@ export MONTAGU_ORDERLY_PATH=$(realpath $here/../src/app/git)
 export ORDERLY_SERVER_USER_ID=$UID
 $here/../scripts/run-dependencies.sh
 
+$here/cli.sh add-user "test.user@example.com"
```

Comment:
Discovered this wasn't working because I wanted to give the test user report reading permissions when running dev dependencies.

Reply:
Oh, I think I just pointed out that was missing in the other branch review!